### PR TITLE
Generation of the stackhci ps cmdlets

### PR DESCRIPTION
**Pull request by lsweeney24**
## Description

Using autorest to automatically create the cmdlets for the stackhci rp. Adding in tests for all cmdlets exposed to a customer.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        -  https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1154
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by VeryEarly** [comment]
"Az.StackHCI","Register-AzStackHCI","Register-AzStackHCI","0","2000","The cmdlet 'Register-AzStackHCI' no longer supports the parameter 'ArcServerResourceGroupName' and no alias was found for the original parameter name.","Add the parameter 'ArcServerResourceGroupName' back to the cmdlet 'Register-AzStackHCI', or add an alias to the original parameter name."
  "Az.StackHCI","Register-AzStackHCI","Register-AzStackHCI","0","2000","The cmdlet 'Register-AzStackHCI' no longer supports the parameter 'ArcSpnCredential' and no alias was found for the original parameter name.","Add the parameter 'ArcSpnCredential' back to the cmdlet 'Register-AzStackHCI', or add an alias to the original parameter name."
  "Az.StackHCI","Register-AzStackHCI","Register-AzStackHCI","0","1050","The parameter set '__AllParameterSets' for cmdlet 'Register-AzStackHCI' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'Register-AzStackHCI'."

are these breaking changes intended? StackHCI is GAed module and breaking changes are not allowed in regular releases

**Comment by priyjain358** [comment]
> "Az.StackHCI","Register-AzStackHCI","Register-AzStackHCI","0","2000","The cmdlet 'Register-AzStackHCI' no longer supports the parameter 'ArcServerResourceGroupName' and no alias was found for the original parameter name.","Add the parameter 'ArcServerResourceGroupName' back to the cmdlet 'Register-AzStackHCI', or add an alias to the original parameter name." "Az.StackHCI","Register-AzStackHCI","Register-AzStackHCI","0","2000","The cmdlet 'Register-AzStackHCI' no longer supports the parameter 'ArcSpnCredential' and no alias was found for the original parameter name.","Add the parameter 'ArcSpnCredential' back to the cmdlet 'Register-AzStackHCI', or add an alias to the original parameter name." "Az.StackHCI","Register-AzStackHCI","Register-AzStackHCI","0","1050","The parameter set '__AllParameterSets' for cmdlet 'Register-AzStackHCI' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'Register-AzStackHCI'."
> 
> are these breaking changes intended? StackHCI is GAed module and breaking changes are not allowed in regular releases

Hi, It's not a breaking change and this powershell script is already released. We are adding management plane cmdlets

File: src/StackHCI/examples/Get-AzStackHciArcSetting.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### Example 1: {{ Add title here }}
```
Comment (dingmeng-xue) [inline src/StackHCI/examples/Get-AzStackHciArcSetting.md:1]: Please remove place holder if you don't want to add title

File: src/StackHCI/examples/Update-AzStackHciArcSetting.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+### Example 1: {{ Add title here }}
+```powershell
```
Comment (dingmeng-xue) [inline src/StackHCI/examples/Update-AzStackHciArcSetting.md:2]: It is empty.

**Review by dingmeng-xue — APPROVED** [review]

**Pull request merged**